src/util.ts

```diff
@@ -41,7 +37,7 @@ const toInt = (value: unknown, defaultValue?: number) => {
     }
   }
 
-  if (typeof value === "undefined") {
+  if (typeof value === "undefined" || value === "") {
```

Comment:
I'm wondering if we should just call `Number` at some point instead of throwing (on line 44)

Reply:
What behavior would you expect if if they pass in a `Number` of 5.5? Would you expect to cast that number to an int (so return `5`?

Reply:
It makes me wonder if we should rename the function (or create a new one that is called `toNumber`). Do we really need an int in most cases?

Reply:
That's a good point. There are definitely times where you want an _int_ but having a `toNumber` is probably a good idea, too. I'll write that up quick.

Reply:
Nice, thanks!